frontend/src/patientApp/timeOfTest/DOB.test.tsx

```diff
@@ -26,13 +27,33 @@ const validateDateOfBirthSpy = jest
   .mockImplementation(jest.fn());
 
 describe("DOB (valid UUID)", () => {
+  let getObfuscatedPatientNameSpy: jest.SpyInstance;
+
   beforeEach(() => {
-    const store = mockStore({
-      plid: uuid(),
-    });
+    const store = mockStore({ plid });
+
+    getObfuscatedPatientNameSpy = jest
+      .spyOn(PxpApi, "getObfuscatedPatientName")
+      .mockImplementation((_plid) => Promise.resolve("John D."));
+
     render(mockContainer(store));
   });
 
+  it("fetches obfuscated patient name from server on render", async () => {
```

Comment:
I dont like using the term `obfuscated`, I think its very misleading since we basically displaying first name and last name initial.

Reply:
Hmm, "obfuscated" was the term used by ISSO (via Tim B) for this. I liked it because it's implementation-independent but maybe there's some other term?